### PR TITLE
Fix brac models failing after first run

### PR DESCRIPTION
**Pull request by lorerod**
closes https://github.com/medic/cht-pipeline/issues/67

### CODE REVIEW

**Comment by lorerod** [comment]
Hi @njuguna-n, I would love to have some BRAC data to see the tables and views with records and compare.

File: models/brac/contact_views/contactview_chp.sql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{
     config(
-        materialized = 'incremental',
+        materialized = 'view',
```
Comment (njuguna-n) [inline models/brac/contact_views/contactview_chp.sql:3]: Is changing this to a view intentional?

**Review by njuguna-n — APPROVED** [review]
Looks good. I have added one question. I have not managed to test locally as I am still testing out another branch.

File: models/brac/contact_views/contactview_chp.sql (hunk 1 above)
Comment (lorerod) [reply models/brac/contact_views/contactview_chp.sql:3]: I noticed this model depended on the contactview_chw."@timestamp" column, but the contactview_chw model doesn't have a @timestamp column. Also, contactview_chw and the other models this depends on are created as views. I don't know how to define this as incremental if we don't have a timestamp to calculate. What do you think?

Comment (lorerod) [reply models/brac/contact_views/contactview_chp.sql]: I didn't find the field `reported` on `contactview_chw,` only on `contactview_metadata,` so I added it and also to `contactview_chp.` Do you think this is ok?

Comment (njuguna-n) [reply models/brac/contact_views/contactview_chp.sql]: That is fine @lorerod.

**Pull request merged**